crates/typst/src/lib.rs

```diff
@@ -26,7 +26,7 @@
 //! [evaluate]: eval::eval
 //! [module]: foundations::Module
 //! [content]: foundations::Content
-//! [layouted]: crate::layout::layout_document
+//! [layout]: crate::layout::layout_document
```

Comment:
This should be `[laid-out]`, to match above, but I can't tell offhand if that's a legal set of characters in this notation.